spec/index.html

```diff
@@ -515,20 +515,99 @@ <h2>Basic Concepts</h2>
     </p>
 
     <p class="ednote">FIXME.</p>
+
+    <section class="informative" id="convenience-context">
+      <h3>Convenience Context</h3>
+      <p>As JSON-LD keywords cannot be represented in YAML without being quoted,
+        publishers may find it useful to declare term aliases for these
+        keywords.</p>
+
+      <p>
+        For convenience, a <dfn>convenience context</dfn> is available
+        at `https://www.w3.org/ns/yaml-ld/v1`, which pre-defines these aliases:
+      </p>
+
+      <pre data-transform="updateExample">
+        <!--
+        {
+          "@context": {
+            "$always": "@always",
+            "$base": "@base",
+            "$container": "@container",
+            "$direction": "@direction",
+            "$embed": "@embed",
+            "$explicit": "@explicit",
+            "$graph": "@graph",
+            "$id": "@id",
+            "$import": "@import",
+            "$included": "@included",
+            "$index": "@index",
+            "$json": "@json",
+            "$language": "@language",
+            "$list": "@list",
+            "$nest": "@nest",
+            "$never": "@never",
+            "$none": "@none",
+            "$null": "@null",
+            "$omitDefault": "@omitDefault",
+            "$once": "@once",
+            "$prefix": "@prefix",
+            "$propagate": "@propagate",
+            "$protected": "@protected",
+            "$requireAll": "@requireAll",
+            "$reverse": "@reverse",
+            "$set": "@set",
+            "$type": "@type",
+            "$value": "@value",
+            "$version": "@version",
+            "$vocab": "@vocab"
+          }
+        }
+        -->
+      </pre>
+
+      <p>The <a>convenience context</a> contains an alias to every JSON-LD keyword which the JSON-LD 1.1
+        specification permits aliasing &mdash; which means all the keywords except <code>@context</code>. The reserved `@` character is
+        replaced by `$`, which is not reserved and therefore does not require quoting. Consider
+        <a href="#example-example-yaml-ld-document-without-convenience-context"></a>
+         reformatted using the <a>convenience context</a>:</p>
+
+    <p>This allows YAML-LD to be created with minimal use of <a>map</a> quoted keys (Note that the `@context` keyword cannot be aliased):</p>
+
+    <pre class="example yaml"
+         data-transform="updateExample"
+         data-content-type="application/ld+yaml"
+         title="Basic YAML-LD document with convenience context">
+      <!--
+      %YAML 1.2
+      ---
+      '@context':
+      - http://example.org/context.jsonld
+      - https://www.w3.org/ns/yaml-ld/v1
+      $graph:
```

Comment:
Does this mean that if I'm offline I cannot even use a JSON-LD processor to understand that `$graph` is a `@graph` ?

Reply:
There is text elsewhere (4.5 Pre-loaded Contexts) that this should be built into the `documentLoader`, so it is always available. It's a SHOULD, but perhaps should be a MUST.

Note that, this is the _default_ `documentLoader`, which can still be updated when invoking the API and it would be up to those implementations to continue to pre-load the context.

Reply:
In the JSON- LD API, it is described here: https://www.w3.org/TR/json-ld11-api/#remote-document-and-context-retrieval. The option to set it is `documentLoader`. It is the general way that documents are retrieved and interpreted.